src/Resources/contao/templates/backend/be_no_root.html5

```diff
@@ -16,8 +16,14 @@ $lang = (object) $GLOBALS['TL_LANG']['XPT'];
   <meta name="viewport" content="width=device-width,initial-scale=1.0">
 
   <style><?php
-    include TL_ROOT . '/system/themes/' . $theme . '/basic.css';
-    include TL_ROOT . '/system/themes/' . $theme . '/error.css';
+    $includeIfExists = function ($file) {
```

Comment:
Depending on the implementation, yes. But my existing adjustments will be gone. It would also require the backend template editor to support Twig etc., so I think this should be in a future version.